docs/howto/deployment/asgi/uvicorn.txt

```diff
@@ -24,12 +24,12 @@ called (separated by a colon).
 
 For a typical Django project, invoking Uvicorn would look like::
 
-    uvicorn myproject.asgi:application
+    gunicorn myproject.asgi:application -k uvicorn.workers.UvicornWorker
 
 This will start one process listening on ``127.0.0.1:8000``. It requires that
 your project be on the Python path; to ensure that run this command from the
 same directory as your ``manage.py`` file.
 
-For more advanced usage, please read the `Uvicorn documentation <Uvicorn_>`_.
+For more advanced usage, please read the `Uvicorn deployment documentation <Uvicorn_>`_.
 
-.. _Uvicorn: https://www.uvicorn.org/
```

Comment:
The deployment guide doesn't cover install and such. I think we should keep the link as is.